src/Tizen.Network.Nfc/Tizen.Network.Nfc/NfcNdefMessage.cs

```diff
@@ -28,7 +28,7 @@ public class NfcNdefMessage : IDisposable
     {
         private bool disposed = false;
         private IntPtr _messageHandle = IntPtr.Zero;
-        private List<NfcNdefRecord> _recordList = new List<NfcNdefRecord>();
+        private ArrayList _recordList = new ArrayList();
```

Comment:
Why don't you use `List<NfcNdefRecord>`

Reply:
Because the List<NfcNdefRecord> can't use the Insert method
-                _recordList.Add(record);
+                _recordList.Insert(index, record);

Reply:
List has Insert method
https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.list-1.insert?view=netframework-4.8

Reply:
remarks of List.
The List<T> class is the generic equivalent of the ArrayList class.

Reply:
I'm sorry about my mistake. I checked that list has insert method. I'll convert the code